Cargo.toml

```diff
@@ -110,8 +110,8 @@ nix = "0.26.2"
 default = ["hunspell", "nlprules"]
 
 # hunspell uses the segmenter provided by nlprules
-hunspell = ["hunspell-rs/bundled", "fd-lock", "nlprules", "encoding_rs"]
-nlprules = ["nlprule/regex-fancy", "nlprule-build"]
+hunspell = ["dep:hunspell-rs", "hunspell-rs?/bundled", "dep:fd-lock", "nlprules", "dep:encoding_rs"]
+nlprules = ["dep:nlprule", "nlprule?/regex-fancy", "dep:nlprule-build"]
```

Comment:
What's the MSRV for this syntax?

Reply:
@drahnr this is available since 1.60 https://doc.rust-lang.org/cargo/reference/features.html